guides/v2.3/graphql/index.md

```diff
@@ -53,3 +53,7 @@ GraphiQL is an in-browser tool for writing, validating, and testing GraphQL quer
 ![GraphiQL browser]({{ page.baseurl }}/graphql/images/graphql-browser.png)
 
 To begin using GraphiQL, set the GraphQL endpoint by entering `http://<magento2-3-server>/graphql` in the URL bar, then click **Set endpoint**. You can use the browser in the right column to determine how to set up a query. Additional examples are also available at [Searches and pagination in GraphQL]({{ page.baseurl }}/graphql/search-pagination.html).
+
+
+{:.bs-callout .bs-callout-info}
+You can access the GraphQL introspection feature only if your Magento instance is in developer mode. See [Set the Magento mode]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-mode.html) for more information about modes.
```

Comment:
Consider leaving off "for more information about modes" --  just 

"See [Set the Magento mode]."

Reply:
That's too terse for me.  Changed to 

[Set the Magento mode] describes how to check and change the mode.